shepherd/bot.py

```diff
@@ -0,0 +1,44 @@
+import json
+import requests
+import Sheet
+import threading
+
+# Set the webhook_url to the one provided by Slack when you create the webhook at https://my.slack.com/services/new/incoming-webhook/
+webhook_url = 'https://hooks.slack.com/services/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' #queuing
+# webhook_url = 'https://hooks.slack.com/services/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' #shepherd-bot-testing
```

Comment:
I've revoked the authorizations for `#queueing` and refreshed the one for `#shepherd-bot-testing`. Remember that PieCentral is public, so sensitive data like this should never be checked into GitHub, not even on a temporary branch.

Slack incoming webhooks don't require authorization because the randomness in the webhook URL provides security. By revealing the random URL, anyone with these URLs could post anything into our Slack as the Shepherd bot.

A better way would be to read from `os.environ` (environment variable) or a config file.